docs/data/material/design-resources/connect/connect.md

```diff
@@ -0,0 +1,64 @@
+# Connect plugin
+
+<p class="description">A Figma plugin that lets you generate a theme from the Material UI Design kit for Figma. </p>
```

Comment:
Yeah, that's a tough one... "Material UI kit" doesn't immediately come across as a Figma version of Material UI to me, though. It could sound like we're referring to possibly another engineering tool, like Apple's UIKit. We could go with something as descriptive and straightforward as "Material UI for Figma" and not even have to rely on "design" or "kit", which are potentially confusing words. Then, this sentence could be:

> Connect is a Figma plugin that lets you generate a theme from Material UI for Figma.

But I'm bothered by repeating Figma twice here 😬 I'll likely just roll with "Material UI Design Kit" for now.